packages/gasket-plugin-docs-graph/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@gasket/plugin-docs-graph",
```

The name for this plugin in your PR title, description, and package name is subtly different. Maybe align on this:
```suggestion
  "name": "@gasket/plugin-docs-graphs",
```